std_msgs/String name_cs
std_msgs/Float64 distance
z_laser_projector/RefPoint p1
z_laser_projector/RefPoint p2
z_laser_projector/RefPoint p3
z_laser_projector/RefPoint p4
z_laser_projector/RefPoint T1
std_msgs/Int16 scale_factor

---

std_msgs/Bool cs_created
std_msgs/String message